app/pages/api/buy.ts

```diff
@@ -0,0 +1,51 @@
+import { NextApiResponse, NextApiRequest } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method !== "POST") {
+    return res.status(404).json({ status: "Not found" });
+  }
+
+  if (!req.body) {
+    return res.status(500).json({ error: "Internal server error" });
+  }
+
+  const reqBody = JSON.parse(req.body);
+
+  if (!reqBody.address) {
+    return res.status(500).json({ error: "Internal server error" });
+  }
+
+  const mobilumUrl = new URL("https://api.mobilum.com/widget/GetWidget");
+  mobilumUrl.searchParams.set("walletAddress", reqBody.address);
+
+  const body = {};
+
+  const widgetResponse = await fetch(mobilumUrl.href, {
+    method: "POST",
+    headers: {
+      ApiKey: process.env.MOBILUM_API_KEY as string,
```

Comment:
I liked your suggestion to create this endpoint and protect this key. But I was poking around the response payload for this service and the key is right there 🙄 so I guess we can move all this logic to the client.
![image](https://user-images.githubusercontent.com/88635679/191602954-27c0bebf-4674-431c-ab1b-16fe462a4b91.png)
I guess these keys are not sensitive. But we will still keep them in `vercel` env vars as a best practice

Reply:
@Atmosfearful OK, I was able to find this finally. Not sure why I couldn't see it before. I have a question into Mobilum about it just so we have an answer.

Would you prefer I just remove the endpoint or should I strip out the `data-apikey` property and continue to send it through.